mava/advanced_usage/ff_ippo_store_experience.py

```diff
@@ -449,6 +448,19 @@ def learner_setup(
     critic_params = critic_network.init(key_p, init_x)
     critic_opt_state = critic_optim.init(critic_params)
 
+    # Load model from checkpoint if specified.
+    if config.logger.checkpointing.load_model:
```

Comment:
Why is checkpointing inside the logger config? I think we can give it its own config? Or make a common base system config with checkpointing inside it?

Reply:
I did it in the logger_setup in order to not broadcast the init params and then we re-do that with checkpointed ones

Reply:
No but the config itself, why does checkpointing live inside logging? If I want to configure checkpointing I shouldn't have to go into the logging folder :thinking:

Reply:
oh I see your point :sweat_smile: do you think it will be better to have separate one?

Reply:
Yeah, I recall having a discussion about it, but unsure why we settled on that location. Can you remember @RuanJohn? Happy for it to go elsewhere, if we're all good with that :)

Reply:
Chatted to @RuanJohn, we're happy to leave as is :smile:

Reply:
sounds great :)